README.md

```diff
@@ -1,3 +1,3 @@
-# Open SDG - Data starter ![Build and Deploy Development Data](https://github.com/CityOfLosAngeles/open-sdg-data-starter/workflows/Build%20and%20Deploy%20Development%20Data/badge.svg) ![Build and Deploy Production Data](https://github.com/CityOfLosAngeles/open-sdg-data-starter/workflows/Build%20and%20Deploy%20Production%20Data/badge.svg)
```

Comment:
Keep the markdown image links so that we can see the workflow status in the default branch.